controllers/module_reconciler.go

```diff
@@ -440,9 +428,55 @@ func (mrh *moduleReconcilerHelper) setKMMOMetrics(ctx context.Context) {
 	err := mrh.client.List(ctx, &mods)
 	if err != nil {
 		logger.V(1).Info("failed to list KMMomodules for metrics", "error", err)
-	}
+		return
+	}
+
+	numModules := len(mods.Items)
+	numModulesWithBuild := 0
+	numModulesWithSign := 0
+	numModulesWithDevicePlugin := 0
+	for _, mod := range mods.Items {
+		signFound := false
+		buildFound := false
+		if mod.Spec.DevicePlugin != nil {
+			numModulesWithDevicePlugin += 1
+		}
+		if mod.Spec.ModuleLoader.Container.Build != nil {
+			buildFound = true
+		}
+		if mod.Spec.ModuleLoader.Container.Sign != nil {
+			signFound = true
+		}
```

Comment:
This is referring only to the global section of the build/sign isn't it? What if there is a build/sign specified only in the kernel mappings?